doc/classes/String.xml

```diff
@@ -669,9 +669,10 @@
 			<argument index="2" name="maxsplit" type="int" default="0">
 			</argument>
 			<description>
-				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				Splits the string by a [code]divisor/delimiter[/code] string and returns an array of the substrings, starting from right.
```

Comment:
`[code]divisor[/code]` refers to the first argument named "divisor". Adding `/delimiter` formatted like an argument name is confusing.

Reply:
Isn't "Divisor" just a description of the variable the user will use? Since Divisor is not any kind of datatype in Godot I think putting code tags on it is confusing. Instead, we could put the code tags around String:
Splits the string by a divisor [code]String[/code] and returns an array of the substrings, starting from the right.
Which is a Valid datatype.

Reply:
Also in the same context, Array should be changed to [code]PoolStringArray[/code].

Reply:
after all, "divisor" as a word is not used by the user at all. It is just there to help the user understand what this string stands for. If i search the docs for "divisor" i'd find nothing, even if i search google "divisor" stands for: "a number by which another number is to be divided.". But "delimiter" on google: "A delimiter is a sequence of one or more characters used to specify the boundary between separate, independent regions in plain text or other data streams"

Reply:
So in the case someone does not unserstand what "divisor" is, they should atleast be able to search for it in google and find an answer.

Reply:
See:
```
			<argument index="0" name="divisor" type="String">
			</argument>
			<argument index="1" name="allow_empty" type="bool" default="True">
			</argument>
			<argument index="2" name="maxsplit" type="int" default="0">
			</argument>
```
`divisor` is the name given to the first argument of this method. Argument names are referred to with `[code]` formatting so that it's clear for the reader that we're referring to the named argument at position 0.
Similarly, we refer to `maxsplit` with `[code]` formatting so that it's clear that we refer to the named argument at position 2.

If you think `divisor` is a confusing name, it should be changed in the method's binding in `core/variant_call.cpp`:
```
        ADDFUNC3R(STRING, POOL_STRING_ARRAY, String, split, STRING, "divisor", BOOL, "allow_empty", INT, "maxsplit", varray(true, 0));
        ADDFUNC3R(STRING, POOL_STRING_ARRAY, String, rsplit, STRING, "divisor", BOOL, "allow_empty", INT, "maxsplit", varray(true, 0));
        ADDFUNC2R(STRING, POOL_REAL_ARRAY, String, split_floats, STRING, "divisor", BOOL, "allow_empty", varray(true));
```
which will also change the name exposed to the docs.

Reply:
Alright cool, then I think we should just change "divisor" to "delimiter" as it is proper naming for this situation. I'll commit the change.